certified-connectors/Encodian/apiDefinition.swagger.json

```diff
@@ -6998,43 +7113,37 @@
           {
             "in": "body",
             "name": "operation",
-            "description": "Request object, refer to schema definition: DtoFormatDate",
+            "description": "Request object, refer to schema definition: DtoValidateUrlSyntax",
             "schema": {
-              "$ref": "#/definitions/DtoFormatDate"
+              "$ref": "#/definitions/DtoValidateUrlSyntax"
             }
           }
         ],
         "responses": {
           "200": {
             "description": "Success",
             "schema": {
-              "$ref": "#/definitions/DtoResponseFormatDate"
-            }
-          },
-          "202": {
-            "description": "Accepted",
-            "schema": {
-              "$ref": "#/definitions/DtoResponseFormatDate"
+              "$ref": "#/definitions/DtoResponseValidateUrlSyntax"
             }
           },
           "400": {
             "description": "Bad request",
             "schema": {
-              "$ref": "#/definitions/DtoResponseFormatDate"
+              "$ref": "#/definitions/DtoResponseValidateUrlSyntax"
             }
           }
         },
         "x-ms-visibility": "important",
-        "summary": "Utility - Format Date",
-        "description": "Format the date value to the format configuration selected - https://support.encodian.com/hc/en-gb/articles/11053469626525"
+        "summary": "Utility - Validate URL Syntax",
+        "description": "Validate whether the URL provided has valid syntax - https://support.encodian.com/hc/en-gb/articles/9601816944413"
       }
     },
-    "/api/v1/Utility/CreateHMACV2": {
+    "/api/v1/Utility/ValidateEmailAddress": {
```

Comment:
Can you please see the below swagger validation error and fix the same?

##[error]The required input field has been added with default value. Existing flows will not send this new field. ValuePath: operation/ValidateEmailAddress/parameter/operation/schema/regex Source: NewSwagger

Reply:
@vmanoharas - we handle this within the API this can be resolved :)